src/views/home/components/EditionGroupCard.vue

```diff
@@ -42,25 +42,25 @@ import { EditionInfo } from '@/models/edition';
 import EditionIcons from '@/components/cues/edition-icons.vue';
 
 @Component({
-    name: 'edition-card',
+    name: 'edition-group-card',
     components: {
         'edition-icons': EditionIcons,
     }
 })
-export default class EditionCard extends Vue {
+export default class EditionGroupCard extends Vue {
+
     @Prop() public edition!: EditionInfo;
 
-    private get lockEdition(): boolean {
-        return this.edition.permission.readOnly;
-    }
     private get thumbnailSource(): string | undefined {
-        return this.edition.thumbnail
-            ? this.edition.thumbnail.thumbnailUrl
-            : undefined;
+        return this.edition?.thumbnail?.thumbnailUrl;
+    }
+
+    private get lockEdition(): boolean {
+        return this.edition!.permission!.readOnly;
     }
 
     private get publicEditionCount(): number {
-        return this.edition.publicCopies;
+        return this.edition!.publicCopies;
```

Comment:
Why is the exclamation point required here? And in other references to this.edition? this.edition is an EditionInfo, it can'tbe null or undefined.